src/test/java/io/openliberty/tools/intellij/it/SingleModMPProjectTestCommon.java

```diff
@@ -782,7 +782,6 @@ public void testMultipleConfigEditHistory() {
     @Test
     @Video
     @EnabledOnOs({OS.LINUX})
```

Comment:
@TrevCraw @turkeylurkey @mrglavas I noticed that these container tests are currently enabled only on Linux. Is there a specific reason for this restriction, or would it be okay if I go ahead and remove it?

Reply:
Linux is the only OS where we were able to set up Docker to enable the devc tests to run.